test/cache_self_test.rb

```diff
@@ -80,6 +80,25 @@ def test_update
     profile.update_attributes(point: 10)
   end
 
+  def test_touch
+    time = Time.now
+    difficulty = Difficulty.create(updated_at: time)
+    assert_queries(1){ assert_equal time.to_i, Difficulty.cacher.find_by(id: difficulty.id).updated_at.to_i }
+    assert_queries(0){ assert_equal time.to_i, Difficulty.cacher.find_by(id: difficulty.id).updated_at.to_i }
+    assert_cache("active_model_cachers_Difficulty_#{difficulty.id}" => difficulty)
+
+    Time.stub :now, Time.at(0) do
+      assert_queries(2){ Difficulty.find(difficulty.id).touch }
```

Comment:
Is there a difference between using `Difficulty.find(difficulty.id).touch` and `difficulty.touch` here?

Reply:
Actually there is not difference. fixed: https://github.com/khiav223577/active_model_cachers/pull/34/commits/bd6e9fcd9a0fca2a32ae5833a98e1bc6eccf6f0d